docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -267,6 +276,53 @@ same project where you build your Quarkus application, everything will work fine
 or jars, you can make sure that your jar is treated like a Quarkus application library by adding an empty `META-INF/beans.xml` file.
 This will allow Quarkus to index and enhance your entities as if they were inside the current project.
 
+== Hibernate ORM in development mode
+
+Quarkus development mode is really useful for applications that mix front end or services and database access.
+There are a few common approaches.
+
+The first choice is to use `quarkus.hibernate-orm.database.generation=drop-and-create` in conjunction with `import.sql`.
+That way for every change to your app and in particular to your entities, the database schema will be properly recreated
+and your data fixture (stored in `import.sql`) will be used to repopulate it from scratch.
+This is best to perfectly control your environment.
+
+The second approach is to use `quarkus.hibernate-orm.database.generation=update`.
+This approach is best when you do many entity changes but
+still need to work on a copy of the production data
+or if you want to reproduce a bug that is based on a specific database entries.
+`update` is a best effort from Hibernate ORM and will fail in specific situations
+including altering your database structure which could lead to data loss.
+For example if you change structures related to the foreign key, Hibernate ORM might have to bail out.
+
+The third approach is to use `quarkus.hibernate-orm.database.generation=none`.
+This approach is best when you are working on a copy of the production data but want to fully control the schema evolution.
+Or if you use a database schema migration tool like Flyway.
```

Comment:
Shouldn't we link to the Quarkus Flyway guide here?

Reply:
+1

Reply:
done